spec/models/pacbio/run_spec.rb

```diff
@@ -16,8 +17,16 @@
       expect(build(:pacbio_sequel_run, sequencing_kit_box_barcode: nil)).to be_valid
     end
 
-    it 'must have a DNA control complex kit box barcode' do
-      expect(build(:pacbio_sequel_run, dna_control_complex_box_barcode: nil)).not_to be_valid
+    context 'v10 and v11' do
+      it 'does not need a DNA control complex kit box barcode' do
```

Comment:
```suggestion
      it 'can have a DNA control complex kit box barcode' do
```

Maybe have a test that `version10` can create a run with or without the `dna_control_complex_box_barcode`, and both be valid.